great_expectations/data_context/data_context.py

```diff
@@ -2756,6 +2761,136 @@ def _list_ymls_in_checkpoints_directory(self):
         files = glob.glob(os.path.join(checkpoints_dir, "*.yml"), recursive=False)
         return files
 
+    def test_yaml_config(
```

Comment:
Does this mean that `test_yaml_config()` will work for DataContext V2?

Reply:
If not, I'm wonder if there would be a clear way to communicate that to the user.  Right now it seems to check if the class name is "Datasource" or "SimpleSqlalchemyDatasource" or else throws an error.

Reply:
@Shinnnyshinshin It will run just fine, but its internal workings will cause it to exit with a friendly message when encountering classes it does not know how to handle (e.g., the non-V3 classes).  This was the existing behavior, but not it does not need to be in a separate class.

Reply:
resolved for me 👍